src/index.html

```diff
@@ -15,8 +15,108 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#moyo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="nav_link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#samsung"
+              class="nav_link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#smartphones"
+              class="nav_link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#laptops&Computers"
+              class="nav_link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#gadgets"
+              class="nav_link"
+            >
+              GADGETS
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#tablets"
+              class="nav_link"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#photo"
+              class="nav_link"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#video"
+              class="nav_link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
```

Comment:
Remember to add empty lines between sibling blocks of HTML to add some 'air' and simplify reading. But don't add them between parent and child elements. In your case, you should add empty lines between `<li>` elements.